pkg/bpf/binary/binary.go

```diff
@@ -1,33 +1,7 @@
+// SPDX-License-Identifier: BSD-3-Clause
 // Copyright Authors of Cilium
-// SPDX-License-Identifier: Apache-2.0
-//
-// Copyright (c) 2009 The Go Authors. All rights reserved.
-//
-// Redistribution and use in source and binary forms, with or without
-// modification, are permitted provided that the following conditions are
-// met:
```

Comment:
I think the intent behind the previous arrangement for this code was that we had relicensed this code Apache-2.0, but it is still subject to the original license. Given the wording here, I don't think we can remove this licensing text.

Reply:
Ah, so it was relicensed after all; I'll change it back. Didn't know if Apache2 and BSD licences were compatible.

The licensing text has been removed in correspondence to the upstream file, see commit msg. The full headers are no longer included.

Reply:
Sure thing! Commit messages are not documentation, so I've added a reference to the path of the original files in the comments.